quelpa.el

```diff
@@ -167,37 +167,6 @@ quelpa cache."
 (defvar quelpa-recipe '(quelpa :repo "quelpa/quelpa" :fetcher github)
   "The recipe for quelpa.")
 
-;; --- compatibility for legacy `package.el' in Emacs 24.3  -------------------
-
-(defun quelpa-setup-package-structs ()
-  "Setup the struct `package-desc' when not available.
-`package-desc-from-legacy' is provided to convert the legacy
-vector desc into a valid PACKAGE-DESC."
-  (unless (functionp 'package-desc-p)
-    (cl-defstruct
-        (package-desc
-         (:constructor
-          ;; convert legacy package desc into PACKAGE-DESC
-          package-desc-from-legacy
```

Comment:
nit: Since you remove this, I guess you can probably remove the check and usage of `package-desc-from-legacy` in line 231-232 too?
Also, remove the `(package-tar-file-info file)` in L227 too? Since from emacs25, that function  has no parameters.

Reply:
Done 6de2811.